src/Message/Message.module.scss

```diff
@@ -0,0 +1,61 @@
+@import '../variables.scss';
+@import '../mixins.scss';
+
+.wrapper {
+  @include text-normal;
+
+  height: 40px;
```

Comment:
If the text is long enough, it will be wrapped to the next line, which breaks the element flow because of the fixed height.
`padding: 8px 12px;` should be used instead